src/phpunit-printer.yml

```diff
@@ -5,10 +5,11 @@ options:
   cd-printer-hide-namespace: true
   cd-printer-anybar: false
   cd-printer-anybar-port: 1738
+  cd-printer-dont-format-classname: false
 markers:
   cd-pass: "✔ "
   cd-fail: "✖ "
   cd-error: "⚈ "
   cd-skipped: "=> "
   cd-incomplete: "∅ "
-  cd-risky: "⌽ "
\ No newline at end of file
+  cd-risky: "⌽ "
```

Comment:
It changed because it's a best practice to always have a carriage return on the last line, i can force my phpstorm to bypass this best practice if you want but it'll not generate a BC break